src/otx/algorithms/visual_prompting/tasks/inference.py

```diff
@@ -406,11 +403,7 @@ def model_info(self) -> Dict:
         Returns:
            Dict: Model info.
         """
-        return {
-            "model": self.model.state_dict(),
-            "config": self.get_config(),
-            "version": self.trainer.logger.version,
-        }
+        return torch.load(self._model_ckpt, map_location="cpu")
```

Comment:
How about adding a code to deal with an edge case where `self._model_ckpt` is None?

Reply:
Done.